code/game/machinery/Sleeper.dm

```diff
@@ -661,7 +664,16 @@
 						playsound(src, 'sound/effects/pop.ogg', 50, 1)
 						H.my_appearance.h_style = "Popped Hair"
 						H.update_hair()
-				else if(isjusthuman(H) && Holiday == APRIL_FOOLS_DAY)
+				else if(isjusthuman(H) && (Holiday == APRIL_FOOLS_DAY || istype(src, /obj/machinery/sleeper/mancrowave/galo)))
```

Comment:
Wouldn't a var be better for this?

Reply:
I wasn't sure what the ideal way to do it was but I changed it to a var.